es6-babel-react-flux-karma/test/components/App.tests.tsx

```diff
@@ -1,10 +1,10 @@
-import * as React from 'react/addons';
+import * as React from 'react';
+import * as TestUtils from 'react-addons-test-utils';
 import App from '../../src/components/App';
 import WhoToGreet from '../../src/components/WhoToGreet';
 import Greeting from '../../src/components/Greeting';
 import GreetingStore from '../../src/stores/GreetingStore';
-
-const { TestUtils } = React.addons;
+React; // use React as an expression to prevent React being purged from dependencies as not used directly
```

Comment:
@johnnyreilly with https://github.com/Microsoft/TypeScript/pull/6290, this should not be needed now. the React import should be preserved regardless. do you want to send a PR for it?